roles/openshift_service_catalog/files/kubesystem_roles_bindings.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kube-system-service-catalog
+objects:
+
+# this isn't being created in the correct namespace....
+- apiVersion: v1
+  kind: Role
+  metadata:
+    name: extension-apiserver-authentication-reader
+    namespace: kube-system
```

Comment:
you need to parameterize this field, even if you never intend to use a different value.  that's why my template did so:
https://github.com/openshift/origin/pull/14630/files#diff-2469e304b4201010f7a68ff63eac6b5bR132

it's due to legacy behavior in the template processor:  if you hardcode a namespace value, we strip it and ignore it.  if you use a parameter, we will respect it.